styles/styles.scss

```diff
@@ -1,12 +1,20 @@
-html {
-    width: 100%;
-    height: 100%;
+:root {
+  --navbar-bg-color: #ffffff;
+  --navbar-link-color: rgba(51, 51, 51, 0.7);
+  --navbar-link-hover-color: #333333;
+  --navbar-link-active-color: #333333;
+  --spacer-small: 0.5rem;
+  --spacer-default: 1rem;
+  --spacer-navbar-y: 3.5rem;
+  --spacer-navbar-x: 11.9375rem;
+  --spacer-big: 1.5rem;
+  --font-size-normal: 1.125rem;
+  --slot-spacer-y: 1rem;
+  --slot-spacer-x: 1.25rem;
```

Comment:
Should use regular --spacer variables, if we need more of them, let's create more, with the right naming convention, and then, use it.